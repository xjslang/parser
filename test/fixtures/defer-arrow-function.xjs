const arrowFunc = () => {
  console.log('arrow function start')
  defer {
    console.log('arrow function defer')
  }
  console.log('arrow function end')
}

arrowFunc()
