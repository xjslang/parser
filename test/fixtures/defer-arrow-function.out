arrow function start
arrow function end
arrow function defer
